modules/ocf_admin/manifests/create/app.pp

```diff
@@ -37,7 +37,7 @@
     # TODO: ideally this file wouldn't be directly readable by staff
     '/etc/ocf-create/ocf-create.conf':
       owner  => create,
-      group  => approve,
+      group  => ocfstaff,
```

Comment:
it's needed by frontends to create so that they can submit tasks to it. that includes approve, atool, and ocfweb currently.